domain-models-interface-extensions/src/test/java/org/folio/rest/tools/GenerateRunnerTest.java

```diff
@@ -126,4 +139,38 @@ public void defaultRamlFilesDir() throws Exception {
     assertTest();
   }
 
+  @Test
+  public void testCreateRamlsLookupList() throws Exception {
+    File src = new File(userDir + "/ramls/");
+    assertTrue(src.exists() && src.isDirectory());
+    File dest = new File(userDir + "/target/ramls/");
+    FileUtils.copyDirectory(src, dest);
+    GenerateRunner.createLookupList(dest, GenerateRunner.RAML_LIST, ".raml");
+    URL ramlListUrl = getClass().getClassLoader().getResource("ramls/" + GenerateRunner.RAML_LIST);
+    assertNotNull(ramlListUrl);
+    List<String> ramls = Files.readAllLines(Paths.get(ramlListUrl.toURI()), StandardCharsets.UTF_8);
+    File[] files = src.listFiles((FileFilter) new SuffixFileFilter(new String[] { ".raml" }, IOCase.INSENSITIVE));
+    assertEquals(files.length, ramls.size());
+    for (File file: files) {
+      assertTrue(ramls.contains(file.getName()));
+    }
+  }
+
+  @Test
+  public void testCreateJsonSchemasLookupList() throws Exception {
+    File src = new File(userDir + "/ramls/");
+    assertTrue(src.exists() && src.isDirectory());
+    File dest = new File(userDir + "/target/ramls/");
+    FileUtils.copyDirectory(src, dest);
+    GenerateRunner.createLookupList(dest, GenerateRunner.JSON_SCHEMA_LIST, ".json", ".schema");
+    URL jsonSchemaListUrl = getClass().getClassLoader().getResource("ramls/" + GenerateRunner.JSON_SCHEMA_LIST);
+    assertNotNull(jsonSchemaListUrl);
+    List<String> jsonSchemas = Files.readAllLines(Paths.get(jsonSchemaListUrl.toURI()), StandardCharsets.UTF_8);
+    File[] files = src.listFiles((FileFilter) new SuffixFileFilter(new String[] { ".json", ".schema" }, IOCase.INSENSITIVE));
+    assertEquals(files.length, jsonSchemas.size());
+    for (File file: files) {
+      assertTrue(jsonSchemas.contains(file.getName()));
+    }
+  }
```

Comment:
These two test methods contain a copy of the code to test, instead test against some expected values.
These two test methods are almost the same, the common code should be extracted.

How about something like this:

    @Test
    public void testCreateRamlsLookupList() {
      assertThat(createLookupList(".raml"),
        containsInAnyOrder("jobs.raml\n", "admin.raml\n", "sample.raml\n"));
    }
  
    @Test
    public void testCreateJsonSchemasLookupList() {
      assertThat(createLookupList(".json", ".schema"),
          containsInAnyOrder(
              "book.schema\n",
              "job.schema\n",
              "bulks.schema\n",
              "last_modified.schema\n",
              "jobs.schema\n",
              "bulk.schema\n",
              "parameter.schema\n",
              "jobs_confs.schema\n",
              "jobs_conf.schema\n"));
    }
  
    private List<String> createLookupList(String... suffixes) {
      try {

Reply:
Makes sense to avoid same lines of code that are to be tested. Will update.